modules/packages/Python.chpl

```diff
@@ -582,12 +638,12 @@ module Python {
       Compile a lambda function like 'lambda x,: x + 1'
 
       Note: this only works with lambdas that accept a tuple of arguments,
-      like 'x,' or '*args'
+      like 'x,' or '\*args'
```

Comment:
That's an interesting addition.  Normally I'd say let's use code-blocks, but this is a no doc'ed function and seems to imply that escaping the `*` is required.  Can you walk me through it?

Reply:
Technically, its not required. I got here because `*` is used in rst/sphinx to bold/italicize text and I had this text elsewhere. This caused build errors (rightly so), and needs to be escaped. But rather than only update the failing cases, I updated everywhere for consistency.

Note that this function is nodoc'd because it is not a part of the public interface and really should not be, BUT the docs are very useful for developers reading the code. Critically, it documents information about how Python references are handled.

Reply:
> But rather than only update the failing cases, I updated everywhere for consistency.

I appreciate you.

> Note that this function is nodoc'd because it is not a part of the public interface and really should not be, BUT the docs are very useful for developers reading the code. Critically, it documents information about how Python references are handled.

Oh yeah, we definitely want to have this information and have it on this function.